### ISSUE TITLE
hyperparameters for resnet50 training

### ISSUE THREAD

**Issue by pengleigithub** [michuanhaohao/AlignedReID#6]
I am wondering if you can share the hyper-parameters for resnet50 training. I have trained a model but it is 2% worse than yours.

**Comment by michuanhaohao** [comment]
@pengleigithub The default settings are the hyper-parameters of the paper. If you directly run the train_alignedreid.py, you will get similar results shown in README.

**Comment by pengleigithub** [comment]
In the paper, it says 'mini-batch size is set to160,in which each identity has 4 images.  Each epoch includes 2000 mini-batches'. I am wondering if there is a typo. Each mini-batch has 40 identities and there are 2000 mini-batches. So there are 40 * 2000 = 80000 identities in each epoch. But none of the datasets has 80000 identities.

**Comment by michuanhaohao** [comment]
@pengleigithub Please wait our new paper Alignedreid++. The experiments of AlignedReID exist some mistakes.

**Comment by godprincelp** [comment]
Can you share the details since it could be a long time before the paper is out?

**Comment by pengleigithub** [comment]
Can you share the details since it might be a long time before the paper is posted? Thx.

**Comment by michuanhaohao** [comment]
AlignedReID++ trains and tests on each datasets, and add some theory explanation.

**Comment by pengleigithub** [comment]
Thx. Can you share the training details on market1501?

**Comment by michuanhaohao** [comment]
All details are in the train_alignedreid.py. There are no any tricks. 
1. Adam
2. 300 epoch
3. lr:0.0002 (0-150); 0.00002 (150-300)
4. loss: softmax + triplet loss
